### PR TITLE
Updated XML format of 5 files

### PR DESCRIPTION
**Pull request by ApCoder123**
ran `mvn xar:format` on each

### CODE REVIEW

File: xwiki-platform-core/xwiki-platform-annotations/xwiki-platform-annotation-ui/src/main/resources/AnnotationCode/AnnotateMenuUIX.xml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<?xml version='1.1' encoding='UTF-8'?>
```
Comment (Enygma2002) [inline xwiki-platform-core/xwiki-platform-annotations/xwiki-platform-annotation-ui/src/main/resources/AnnotationCode/AnnotateMenuUIX.xml]: This does not look right. Should use double quotes, for consistency at least.

File: xwiki-platform-core/xwiki-platform-annotations/xwiki-platform-annotation-ui/src/main/resources/AnnotationCode/AnnotateMenuUIX.xml [hunk 2]
```diff
@@ -20,22 +20,22 @@
...
-  <date>1447861877000</date>
-  <contentUpdateDate>1447859231000</contentUpdateDate>
+  <author>XWiki.admin</author>
+  <contentAuthor>XWiki.admin</contentAuthor>
+  <date>1514830377000</date>
+  <contentUpdateDate>1514830377000</contentUpdateDate>
   <version>1.1</version>
   <title/>
-  <comment/>
+  <comment>Install extension [org.xwiki.platform:xwiki-platform-annotation-ui/9.11]</comment>
```
Comment (Enygma2002) [inline xwiki-platform-core/xwiki-platform-annotations/xwiki-platform-annotation-ui/src/main/resources/AnnotationCode/AnnotateMenuUIX.xml]: Are you sure you ran `xar:format`? Comments should not be included in the committed page.

File: xwiki-platform-core/xwiki-platform-annotations/xwiki-platform-annotation-ui/src/main/resources/AnnotationCode/AnnotateMenuUIX.xml [hunk 3]
```diff
@@ -20,22 +20,22 @@
...
 
-<xwikidoc version="1.2" reference="AnnotationCode.AnnotateMenuUIX" locale="">
+<xwikidoc version="1.3" reference="AnnotationCode.AnnotateMenuUIX" locale="">
   <web>AnnotationCode</web>
   <name>AnnotateMenuUIX</name>
   <language/>
   <defaultLanguage/>
   <translation>0</translation>
-  <creator>xwiki:XWiki.Admin</creator>
+  <creator>XWiki.admin</creator>
```
Comment (Enygma2002) [inline xwiki-platform-core/xwiki-platform-annotations/xwiki-platform-annotation-ui/src/main/resources/AnnotationCode/AnnotateMenuUIX.xml]: This should not be changed.

File: xwiki-platform-core/xwiki-platform-annotations/xwiki-platform-annotation-ui/src/main/resources/AnnotationCode/AnnotateMenuUIX.xml [hunk 4]
```diff
@@ -20,22 +20,22 @@
...
   <creationDate>1447859230000</creationDate>
   <parent>AnnotationCode.WebHome</parent>
-  <author>xwiki:XWiki.Admin</author>
-  <contentAuthor>xwiki:XWiki.Admin</contentAuthor>
-  <date>1447861877000</date>
-  <contentUpdateDate>1447859231000</contentUpdateDate>
+  <author>XWiki.admin</author>
```
Comment (Enygma2002) [inline xwiki-platform-core/xwiki-platform-annotations/xwiki-platform-annotation-ui/src/main/resources/AnnotationCode/AnnotateMenuUIX.xml]: Same.

File: xwiki-platform-core/xwiki-platform-annotations/xwiki-platform-annotation-ui/src/main/resources/AnnotationCode/AnnotateMenuUIX.xml [hunk 5]
```diff
@@ -20,22 +20,22 @@
...
+  <contentAuthor>XWiki.admin</contentAuthor>
```
Comment (Enygma2002) [inline xwiki-platform-core/xwiki-platform-annotations/xwiki-platform-annotation-ui/src/main/resources/AnnotationCode/AnnotateMenuUIX.xml]: Same.

**Comment by ApCoder123** [comment]
@Enygma2002 I fixed all the errors. It is very strange, as I definitely ran `mvn xar:format`

**Comment by ApCoder123** [comment]
@Enygma2002 I finally managed to get `mvn xar:format` to work. All the files are now correctly updated

**Pull request merged**